src/containers/home/Home.test.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable testing-library/prefer-screen-queries */
+/* eslint-disable testing-library/render-result-naming-convention */
+
+import { render, fireEvent, screen } from "@testing-library/react";
+import { Provider } from "react-redux";
+import { BrowserRouter } from "react-router-dom";
+
+import Home from "./Home";
+import store from "../../store/store";
+
+import { AuthContext } from "../auth/AuthProvider";
+import { checkProps, getByTestId } from "../../utils/TestUtils";
+import * as userSelector from "../../store/selectores/user";
+
+const defaultProps = {};
```

Comment:
since Home component not accept the props no need to define defaultProps here

Reply:
Fixed